cmd/query.go

```diff
@@ -18,14 +19,28 @@ import (
 	"github.com/turbot/steampipe/workspace"
 )
 
-// QueryCmd :: represents the query command
-func QueryCmd() *cobra.Command {
+// queryCmd :: represents the query command
+func queryCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:              "query",
 		TraverseChildren: true,
 		Args:             cobra.ArbitraryArgs,
 		Run:              runQueryCmd,
-		Short:            "Execute SQL queries interactively or by argument",
+		ValidArgsFunction: func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+			workspace, err := workspace.Load(viper.GetString(constants.ArgWorkspace))
+			if err != nil {
+				return []string{}, cobra.ShellCompDirectiveError
+			}
+			defer workspace.Close()
+			namedQueries := []string{}
+			for key := range workspace.GetNamedQueryMap() {
```

Comment:
shouldn't we sort these as we do for controls - let's add a workspace function to return a list or sorted named queries